code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -171,7 +172,7 @@
 				dropItemToGround(belt)
 		w_uniform = null
 		update_suit_sensors()
-		if(!QDELETED(src)) 
+		if(!QDELETED(src))

```

Comment:
I'm not actually sure why this happens in basically all of my PRs. I believe it's either Dream Maker or TortoiseGit being picky and trying to make all the code look perfect.